.gitignore

```diff
@@ -0,0 +1,2 @@
+.idea*
+.DS_Store
```

Comment:
I put these in my global git ignore as they feel environment specific, rather than project specific.

Reply:
I think they are common enough to justify being per project, 2 lines in a .gitignore is pretty cheap. Also having these in a global .gitignore is a good idea though.

Reply:
Yip, don't mind, just mentioning.

Reply:
Some useless input: I like to use [this gitignore](https://github.com/github/gitignore/blob/master/Global/JetBrains.gitignore) for personal projects or ones on which I know other devs will be using JetBrains IDEs